src/zkevm_specs/util/gas.py

```diff
@@ -0,0 +1,41 @@
+from ..encoding import U8, U64, U128, U256
+from typing import Sequence, Tuple
+
+G_MEM = 3
+
+
+def memory_size(
+    address: U64,
+) -> U64:
+    return (address + 31) // 32
+
+
+def div(
+    value: U256,
+    divisor: U64,
+) -> Tuple[U256, U256]:
+    quotient = value // divisor
+    remainder = value % divisor
+    return (quotient, remainder)
+
+
+def memory_expansion(
```

Comment:
No actions for the PR, just a note for other reviewers:
This part of the code seems duplicated with the one in [mload_mstore](https://github.com/appliedzkp/zkevm-specs/blob/28b299fed41e8a1180aef8033290c94702ff649f/src/zkevm_specs/opcode/mload_mstore.py#L78). There's also a similar one in the `instruction.py`.
IIUC, unlike the other places, where the memory_expansion is used as circuit code, this particular memory_expansion function is only used as utilities in the tests.
We could merge them together somehow in other future PR.

Reply:
yes,  but method in `instruction.py` can not use in the test module,  adding it into `util` enable all related tests can take use of it directly.

Reply:
makes sense, the current approach sounds good.